websites/api-docs-smoke-test/src/generated-api-specs/test/types/EnumPropertiesType.raml

```diff
@@ -6,23 +6,35 @@ description: |-
   Test data for properties with enums that are of differing primitive types
 properties:
   intWithEnumExample:
-    format: int16
+    format: float
     type: number
     (builtinType): number
     description: |-
       An int with enum values
+    enum:
+    - 1
+    - 2
+    - 3
```

Comment:
Something wrong here?  The test case's name is explicitly "an int with enum values" but the generated RAML declares a number with float format.

Reply:
The issue is coming from the input raml.

The type is defined as number without any format. And number defaults to float. So either declare the type to be integer or add a format.

Reply:
Test RAML bug number 10 or so.  It's scary how easy it is / was to write wrong RAML without noticing.

Reply:
Nah any integer is actually a valid float. So it's not necessarily wrong ;)

Reply:
Let's not start that discussion here  :-)